.github/workflows/tx-pull.yml

```diff
@@ -9,13 +9,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
     - name: Install Transifex client
       run: |
         curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
         chmod +x ./tx
     - name: Pull translations from Transifex
       run: |
-        ./tx -t ${{ secrets.TX_TOKEN }} pull -a -f
+        ./tx -t ${{ secrets.TX_TOKEN }} pull -a -t -f
```

Comment:
To fix the broken run: https://github.com/ipfs/ipfs-webui/actions/runs/3257699721